Detections/Tenable/Tenable - Med Vulnerability found.yaml

```diff
@@ -0,0 +1,20 @@
+id: 9d7b2b02-8b44-4850-89e9-a9ca442bc834
+name: Tenable - Medium Vulnerability found
+description: |
+  'This query identifies a Tenable Vulnerability of Medium Vulnerability.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Tenable
+    dataTypes: 
+      - Tenable_Assets_List
+queryFrequency: 12h
+queryPeriod: 12h
+triggerOperator: gt
+triggerThreshold: 0
+query: |
+
+  Tenable_Assets_List
+  | join (Tenable_Vulns_List) 
+      on AssetID
+  | project Timestamp=TimeGenerated1, AssetID, FQDN, IP, Operating_system, VulnID, VulnName, plugin_family, SeverityType, Detection_source
+  | where SeverityType in ("Medium")
```

Comment:
Generally you will want the where clause above the project for performance reasons.  I don't see any reason to NOT move it above the project here.